src/include/connection.h

```diff
@@ -456,16 +456,17 @@ struct __wt_connection_impl {
 	 * delays have been requested.
 	 */
 /* AUTOMATIC FLAG VALUE GENERATION START */
-#define	WT_TIMING_STRESS_CHECKPOINT_SLOW	0x001u
-#define	WT_TIMING_STRESS_LOOKASIDE_SWEEP	0x002u
-#define	WT_TIMING_STRESS_SPLIT_1		0x004u
-#define	WT_TIMING_STRESS_SPLIT_2		0x008u
-#define	WT_TIMING_STRESS_SPLIT_3		0x010u
-#define	WT_TIMING_STRESS_SPLIT_4		0x020u
-#define	WT_TIMING_STRESS_SPLIT_5		0x040u
-#define	WT_TIMING_STRESS_SPLIT_6		0x080u
-#define	WT_TIMING_STRESS_SPLIT_7		0x100u
-#define	WT_TIMING_STRESS_SPLIT_8		0x200u
+#define	WT_TIMING_STRESS_AGGRESSIVE_SWEEP   0x001u
```

Comment:
Indentation doesn't look correct. I suspect that you edited these values by hand. An easier way is to let them autogenerate. Look at `dist/flags.py` to understand how to get them autogenerating:

`To add a new flag declare it at the top of the flags list as:                  
#define WT_NEW_FLAG_NAME      0x0u`